index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<link rel="stylesheet" href="styles.css">
+	<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+	<div class="about-container">
+    <header class="menu-bar">
+      <img src="img/My-Logo.svg" alt="logo">
+      <img src="img/Union.svg" alt="Hamburger">
+    </header>
```

Comment:
- [ ] Nice we have the logo on the left side, however it has a link (a tag), please use a tag instead of image.